packages/appeals-service-api/src/db/seed/data-dev.js

```diff
@@ -61,22 +118,167 @@ const appeals = [
 const appealCases = [
 	{
 		Appeal: {
-			connect: { id: appealOne.id }
+			connect: { id: appealIds.appealOne }
 		},
-		caseReference: '1010101',
+		caseReference: caseReferences.caseReferenceOne,
 		LPACode: 'Q9999',
 		LPAName: 'System Test Borough Council',
 		appealTypeCode: 'HAS',
 		appealTypeName: 'Householder',
-		decision: 'refused',
-		originalCaseDecisionDate: pickRandom(pastDates),
+		decision: '',
+		originalCaseDecisionDate: pickRandom(datesNMonthsAgo(1)),
+		costsAppliedForIndicator: false,
+		LPAApplicationReference: '12/2323231/PLA',
+		siteAddressLine1: '123 Fake Street',
+		siteAddressTown: 'Testville',
+		siteAddressCounty: 'Countyshire',
+		siteAddressPostcode: 'BS1 6PN',
+		questionnaireDueDate: pickRandom(datesNMonthsAhead(1)),
+		interestedPartyRepsDueDate: pickRandom(datesNMonthsAhead(1))
+	},
+	{
+		Appeal: {
+			connect: { id: appealIds.appealTwo }
+		},
+		caseReference: caseReferences.caseReferenceTwo,
+		LPACode: 'Q9999',
+		LPAName: 'System Test Borough Council',
+		appealTypeCode: 'HAS',
+		appealTypeName: 'Householder',
+		decision: '',
+		originalCaseDecisionDate: pickRandom(datesNMonthsAgo(2)),
 		costsAppliedForIndicator: false,
 		LPAApplicationReference: '12/2323232/PLA',
 		siteAddressLine1: '123 Fake Street',
 		siteAddressTown: 'Testville',
 		siteAddressCounty: 'Countyshire',
 		siteAddressPostcode: 'BS1 6PN',
-		questionnaireDueDate: pickRandom(futureDates)
+		questionnaireDueDate: pickRandom(datesNMonthsAhead(2)),
+		interestedPartyRepsDueDate: pickRandom(datesNMonthsAhead(2))
+	},
+	{
+		Appeal: {
+			connect: { id: appealIds.appealThree }
+		},
+		caseReference: caseReferences.caseReferenceThree,
+		LPACode: 'Q9999',
+		LPAName: 'System Test Borough Council',
+		appealTypeCode: 'HAS',
+		appealTypeName: 'Householder',
+		decision: '',
+		originalCaseDecisionDate: pickRandom(datesNMonthsAgo(1)),
+		costsAppliedForIndicator: false,
+		LPAApplicationReference: '12/2323233/PLA',
+		siteAddressLine1: '123 Fake Street',
+		siteAddressTown: 'Testville',
+		siteAddressCounty: 'Countyshire',
+		siteAddressPostcode: 'BS1 6PN',
```

Comment:
Shall we pull out these common bits so we can spread them in?

Reply:
done!